README.md

```diff
@@ -398,23 +398,26 @@ More information here: [Environment variables](https://docs.appdynamics.com/22.2
 
 Please note that AppDynamics requires Mendix 6.2 or higher.
 
-| Environment Variable | Value Example | Description |
+| Environment Variable | Default | Description |
 |-|-|-|
 | `APPDYNAMICS_CONTROLLER_PORT` | 443 | Port of AppDynamics controller |
 | `APPDYNAMICS_CONTROLLER_SSL_ENABLED` | true | Set if SSL is required |
-| `APPDYNAMICS_CONTROLLER_HOST_NAME` | account-name.saas.appdynamics.com | Name of AppDynamics host without 'http://' |
-| `APPDYNAMICS_AGENT_APPLICATION_NAME` | myapp | Name of the app to be displayed on Controller UI |
-| `APPDYNAMICS_AGENT_ACCOUNT_NAME` | account-name | See 'License/Account' on the Controller UI |
-| `APPDYNAMICS_AGENT_ACCOUNT_ACCESS_KEY` | secret_key | See 'License/Account' on the Controller UI |
-| `APPDYNAMICS_AGENT_NODE_NAME` * | mynode | How a node is displayed on the Controller UI |
-| `APPDYNAMICS_AGENT_TIER_NAME` | mytier | How a tier is displayed on the Controller UI |
+| `APPDYNAMICS_CONTROLLER_HOST_NAME` | - | Name of AppDynamics host without 'http://' |
+| `APPDYNAMICS_AGENT_APPLICATION_NAME` | App name as on Dev. Portal | Name of the app to be displayed on Controller UI |
+| `APPDYNAMICS_AGENT_ACCOUNT_NAME` | - | See 'License/Account' on the Controller UI |
+| `APPDYNAMICS_AGENT_ACCOUNT_ACCESS_KEY` | - | See 'License/Account' on the Controller UI |
+| `APPDYNAMICS_AGENT_NODE_NAME` * | node | How a node is displayed on the Controller UI |
```

Comment:
Would `node` be the hostname ? Maybe an example would help.

Reply:
Added a description and example column